cosmo_tester/test_suites/test_simple_manager_blueprint/abstract_single_host_test.py

```diff
@@ -0,0 +1,135 @@
+__author__ = 'maxim'
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+import os
+
+from fabric import api as fabric_api
+
+from cloudify.workflows import local
+from cloudify_cli import constants as cli_constants
+from cosmo_tester.framework.git_helper import clone
+from cosmo_tester.framework.util import create_rest_client
+
+MANAGER_BLUEPRINTS_REPO_URL = 'https://github.com/cloudify-cosmo/' \
+                              'cloudify-manager-blueprints.git'
+
+
+class AbstractSingleHostTest(object):
+
+    def abstract_setup(self):
```

Comment:
maybe use a more explicit name? something like "setup_simple_manager_env"?
this will make it clearer when the actual tests call `self.setup_simple_manager_env()" instead of the more vague `self.abstract_setup()`